### ISSUE TITLE
BarLoader will not appear on v0.10.4

### ISSUE THREAD

**Issue by TrySickle** [davidhu2000/react-spinners#341]
**Describe the bug**
BarLoader will show on v0.9.0 but not after upgrading to v0.10.4. It seems to have 0 width.
The code that worked on v0.9.0 was this:
`<BarLoader
            css={{ margin: '0 auto' }}
            height={ 10 }
            width={ 300 }
            color="#4285F4"
            loading
          />`

**To Reproduce**
1. Upgrade to v0.10.4
2. BarLoader no longer appears

**Screenshots**
![image](https://user-images.githubusercontent.com/8213689/103578325-31ed7b80-4e8b-11eb-9181-bfba28e1cb56.png)

**Additional context**
Still using the class version with react 17

**Comment by filipmacek** [comment]
Also not working for me

**Comment by davidhu2000** [comment]
looks like it's due to the `div` to `span` change, try adding a override for `display: inherit`

**Comment by TrySickle** [comment]
nice, that fixed it

**Comment by davidhu2000** [comment]
probably going to add `display:inherit` to all the loaders

**Comment by 55Cancri** [comment]
@davidhu2000 tried using the latest version and adding `display:inherit` but the component is still broken and does not show.

**Comment by ckeeney** [comment]
Can someone provide a complete example of how to render the `BarLoader` component?

This code does not work.
```js
                <div
                    sx={{
                        display: 'block',
                        backgroundColor: 'green',
                        height: 40,
                    }}
                >
                    <LoadingIndicator
                        height={10}
                        width={100}
                        css={{
                            display: 'inherit',
                            backgroundColor: 'salmon',
                            margin: '0 auto',
                        }}
                        color="#4285F4"
                        loading
                    />
                </div>
```

**Comment by Nate-Wessel** [comment]
I just spent half an hour running in circles until I found this issue. The first time I used this loader, I added `display: inline-block;` explicitly and it worked perfectly. The second time, I copied that code, removing the unneeded CSS. Couldn't figure out for the life of my why it wasn't displaying with nothing else changed.

**Comment by davidhu2000** [comment]
should be fixed as part of 0.13.0-alpha.4
